modules/sdk-core/src/bitgo/tss/ecdsa/ecdsa.ts

```diff
@@ -181,8 +184,8 @@ export async function sendShareToBitgo(
   walletId: string,
   txRequestId: string,
   shareType: SendShareType,
-  share: SShare | MUShare | KShare,
-  dShare?: DShare
+  share: SShare | MuDShare | KShare,
```

Comment:
NIT: can you update the function params description also with the change

Reply:
we can but I don't want to hold up this pr